paymentsheet/src/main/java/com/stripe/android/paymentsheet/paymentdatacollection/cvcrecollection/CvcRecollectionScreen.kt

```diff
@@ -32,12 +38,37 @@ import com.stripe.android.model.CardBrand
 import com.stripe.android.paymentsheet.R
 import com.stripe.android.ui.core.elements.CvcController
 import com.stripe.android.ui.core.elements.CvcElement
+import com.stripe.android.ui.core.elements.H4Text
 import com.stripe.android.uicore.StripeTheme
 import com.stripe.android.uicore.elements.IdentifierSpec
 import com.stripe.android.uicore.elements.SectionCard
+import com.stripe.android.uicore.getBackgroundColor
 import com.stripe.android.uicore.stripeColors
 import com.stripe.android.uicore.utils.stateFlowOf
 
+@Composable
+internal fun CvcRecollectionScreen(cardBrand: CardBrand, lastFour: String) {
+    val element = remember {
+        CvcElement(IdentifierSpec(), CvcController(cardBrandFlow = stateFlowOf(CardBrand.Unknown)))
+    }
+
+    StripeTheme {
+        Column(
+            Modifier
+                .background(Color.White)
```

Comment:
We'll need to update this background color to be `MaterialTheme.stripeColors.surface` to make sure this works for all `Appearance` modes. Will need to regenerate the screenshot tests after as well.

Reply:
Updated 👍